### PR TITLE
Moved member variables to initializer list

### PR DESCRIPTION
**Pull request by Rubonnek**

### CODE REVIEW

File: drivers/windows/semaphore_windows.cpp [hunk 1]
```diff
@@ -70,23 +70,23 @@ void SemaphoreWindows::make_default() {
...
+SemaphoreWindows::SemaphoreWindows() :
 #ifdef UWP_ENABLED
-	semaphore = CreateSemaphoreEx(
-			NULL,
-			0,
-			0xFFFFFFF, //wathever
-			NULL,
-			0,
-			SEMAPHORE_ALL_ACCESS);
+		semaphore(CreateSemaphoreEx(
```
Comment (akien-mga) [inline drivers/windows/semaphore_windows.cpp]: I think this change makes things harder to read, both with the `#ifdef` in the middle of the initializer list and the long function call. Doesn't seem worth the change IMO.

Comment (Rubonnek) [reply drivers/windows/semaphore_windows.cpp]: I understand. Change dropped.

File: drivers/windows/mutex_windows.cpp [hunk 2]
```diff
@@ -75,11 +75,13 @@ void MutexWindows::make_default() {
 	create_func = create_func_windows;
 }
 
-MutexWindows::MutexWindows() {
-
+MutexWindows::MutexWindows()
 #ifdef WINDOWS_USE_MUTEX
-	mutex = CreateMutex(NULL, FALSE, NULL);
-#else
+		:
```
Comment (akien-mga) [inline drivers/windows/mutex_windows.cpp]: This one also looks weird, I'd prefer to keep the previous code.

Comment (Rubonnek) [reply drivers/windows/mutex_windows.cpp]: No problemo. Change undone.

**Pull request merged**